master/reference/typha/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring Typha
+canonical_url: 'https://docs.projectcalico.org/v3.1/reference/typha/configuration'
+---
+
+Configuration for Typha is read from one of two possible locations, in
+order, as follows.
+
+1.  Environment variables, prefixed with `TYPHA_`.
+
+2.  The Typha configuration file.  The path to this file defaults to
+    `/etc/calico/typha.cfg` but can be overridden using the `-c` or
+    `--config-file` options on the command line.
+
+The value of any configuration parameter is the value read from the
+*first* location containing a value. For example, if an environment variable
+contains a value, it takes precedence.
+
+If not set in any of these locations, most configuration parameters have
+defaults, and it should be rare to have to explicitly set them.
+
+The full list of parameters which can be set is as follows.
+
+#### General configuration
+
+| Configuration parameter           | Environment variable                    | Description  | Schema |
+| --------------------------------- | --------------------------------------- | -------------| ------ |
+| `DatastoreType`                   | `TYPHA_DATASTORETYPE`                   | The datastore that Typha should read endpoints and policy information from. [Default: `etcdv3`] | `etcdv3`, `kubernetes`|
+| `HealthEnabled`                   | `TYPHA_HEALTHENABLED`                   | When enabled, exposes typha health information via an http endpoint. | boolean |
```

Comment:
I think here also it should be capitalized: "Typha" (in the description)

Reply:
Sure.  Again, I think it would then make sense to capitalize the corresponding Felix things.